FinalEngine.Tests/Editor/ViewModels/Docking/Tools/Mocks/MockToolViewModel.cs

```diff
@@ -0,0 +1,11 @@
+// <copyright file="MockToolViewModel.cs" company="Software Antics">
+// Copyright (c) Software Antics. All rights reserved.
+// </copyright>
+
+namespace FinalEngine.Tests.Editor.ViewModels.Docking.Tools.Mocks;
+
+using FinalEngine.Editor.ViewModels.Docking.Tools;
+
+public sealed class MockToolViewModel : ToolViewModelBase
```

Comment:
Add code docs here?

Reply:
By preference, I choose not to provide documentation in the test project. Makes for writing tests allot easier. I have StyleCop configurations for my test project to not notify me to provide documentation.